src/checkbox.styles.ts

```diff
@@ -13,27 +13,23 @@ export default [
       line-height: 100%;
 
       &.large {
-        column-gap: var(--glide-core-spacing-sm);
-        font-family: var(--glide-core-body-sm-font-family);
-        font-size: var(--glide-core-body-sm-font-size);
-        font-style: var(--glide-core-body-sm-font-style);
-        font-variant: var(--glide-core-body-sm-font-variant);
```

Comment:
Similar to `font-style`, we use the default `font-variant`, so no need to be explicit.